charms/worker/k8s/lib/charms/k8s/v0/k8sd_api_manager.py

```diff
@@ -49,6 +50,18 @@ class utilises different connection factories (UnixSocketConnectionFactory
 logger = logging.getLogger(__name__)
 
 
+class ErrorCodes(enum.Enum):
+    """Enumerate the response codes from the k8s api.
+
+    Attributes:
+        StatusNodeUnavailable: returned when the node isn't in the cluster
+        StatusNodeInUse: returned when the node is in the cluster already
+    """
+
+    StatusNodeUnavailable = 520
+    StatusNodeInUse = 521
+
+
```

Comment:
These will come in handy once this is available in the snap
* https://github.com/canonical/k8s-snap/pull/388

they aren't necessary but they are a much better indication of whether a unit is already in the cluster, or not yet in the cluster